docs/snapshots.md

```diff
@@ -1,44 +1,47 @@
 
-![snapshot](assets/img/snapshot.png)
+### Snapshots in Your Org
+Snapshots and [Tilt Teams](teams.html) are two great tastes that taste great together! When running a Tiltfile associated with a team (via the `set_team` functions), any snapshots you create are *private*, and are only accessible by members of that team. You can use snapshots with confidence that any sensitive information in your logs is safe from prying eyes.
 
-## Creating and sharing private team snapshots
+Snapshots are a great tool for debugging **remotely** and **asynchronously**--two necessities in the current age of work-from-home. Snapshots eliminate the need for Slack back-and-forth when diagnosing a problem; instead of guessing what error message to copy and paste, a developer can just send their entire Tilt state. Likewise, the frozen-in-time nature of Snapshots makes async debugging a breeze: if a dev encounters strange behavior and the right person to ask isn't online, it's easy to take a snapshot, send over the link, and go back to writing code. 
 
-[Ensure your project is associated with a Tilt Cloud team using `set_team` in your Tiltfile.](teams.html) Make sure you are either a [`Member` or `Owner`](teams.html#add-users-to-your-team) of the team.
-
-Inside Tilt, click the `Create a snapshot` button at the top. If you haven't yet connected Tilt to Tilt Cloud, you'll be prompted to do so (using your GitHub account to sign in). Click `GET LINK` to create the snapshot. Click `OPEN` and Tilt will open the snapshot in Tilt Cloud in a new browser tab. Copy the URL of the snapshot and share it with other team members (via Slack or email, for example). The snapshot is only accessible to Tilt Cloud signed-in users who are a member (or owner) of the team
-
-**Important**: If the Tiltfile errors on its first execution when you run `tilt up`, and [`set_team`](https://docs.tilt.dev/api.html#api.set_team) has not been called yet, Tilt will not understand `set_team`. If you then take a snapshot, it will be a _public_ snapshot. (See below for public snapshots.) To work around this problem, make sure to put `set_team` as the first line in your Tiltfile. Tilt will then successfully parse `set_team` and start using the team id in it. Create a snapshot as normal and it will be a private team snapshot.
+## Creating and Sharing Snapshots
+Inside Tilt, click the `Create a snapshot` button at the top. If you haven't yet connected Tilt to Tilt Cloud, you'll be prompted to do so (using your GitHub account to sign in). Click `GET LINK` to create the snapshot. Click `OPEN` and Tilt will open the snapshot in Tilt Cloud in a new browser tab. Copy the URL of the snapshot and share at will.
 
+// TOOD: maybe a gif here instead?
 ![create-snapshot](assets/img/create-snapshot.png)
 
-## Viewing all private team snapshots
+Visit [cloud.tilt.dev/snapshots](https://cloud.tilt.dev/snapshots) for a list of all active snapshots you have created.
```

Comment:
```suggestion
Visit [cloud.tilt.dev/snapshots](https://cloud.tilt.dev/snapshots) for a list of all snapshots you have created.
```

Let's remove "active" since there's no notion of active vs inactive for snapshots.